setup.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup
+
+
+setup(
+    name='eegrasp',
+    version='0.0.2',
+    description='Graph Signal Processing in Python',
+    long_description=open('README.md').read(),
+    long_description_content_type='text/x-rst',
+    author='GSP EEG',
+    url='https://github.com/gsp-eeg/EEGrasp',
+    project_urls={
+        'Documentation': 'https://eegrasp.readthedocs.io',
+        'Download': 'https://pypi.org/project/eegrasp',
+        'Source Code': 'https://github.com/gsp-eeg/eegrasp',
+        'Bug Tracker': 'https://github.com/gsp-eeg/eegrasp/issues',
+        #'Try It Online': 'https://mybinder.org/v2/gh/epfl-lts2/eegrasp/master?urlpath=lab/tree/examples/playground.ipynb',
+    },
+    packages=[
+        'eegrasp',
+        'eegrasp.tests',
+    ],
+    package_data={'eegrasp': ['data/pointclouds/*.mat']},
+    test_suite='eegrasp.tests.suite',
+    python_requires='>=3.7',
```

Comment:
I think we mentioned 3.11 in other places.

Reply:
Fixed